Documentation/coding-guidelines/clr-code-guide.md

```diff
@@ -8,7 +8,85 @@ Written in 2006, by:
 - Mike Stall ([@mikestall](https://github.com/mikestall))
 - Rudi Martin ([@Rudi-Martin](https://github.com/Rudi-Martin))
 
-# Why you must read this document
+# Contents
+
+* [1 Why you must read this document](#1)
+  * [1.1 Rules of the Code](#1.1)
+  * [1.2 How do I &lt;insert common task&gt;?](#1.2)
+* [2 Rules of the Code (Unmanaged)](#2)
```

Comment:
> at least in technical books

I believe proper use of GitHub flavored markdown is more critical here.

> Also the style I used is also used in other parts of the documentation.

I couldn't find custom-nested ordered-list usage in CoreCLR docs, but IMO those should be changed to standard markdown syntax as well.